steam_wrapper/steam_wrapper.py

```diff
@@ -258,4 +288,8 @@ def main(steam_binary=STEAM_PATH):
     timezone_workaround()
     configure_shared_library_guard()
     enable_discord_rpc()
-    os.execve(steam_binary, [steam_binary] + sys.argv[1:], os.environ)
+    n = os.fork()
+    if n == 0:
```

Comment:
I don't like this. If you want to keep running, remove execve and instead call steam using subprocess. Fork without exec is not really safe.

Reply:
My latest commit now this does, marking as resolved.